src/app/modules/project/add/manual/manual.component.html

```diff
@@ -84,6 +83,30 @@ <h3>Project icon</h3>
       </div>
     </form>
 
+    <form [formGroup]="newCallToActionForm" class="project-form">
```

Comment:
In case there are no call to actions, we should hide this form as it only creates confusion for the person adding a project.
A simple:
<ng-container *ngIf="callToActionOptions.length !== 0"> or something will do the trick probably.

Reply:
Please note that this feedback is applicable to editing a project as well.